### PR TITLE
Document quantile and quantile_over_time

### PR DESCRIPTION
**Pull request by brian-brazil**
Goes with https://github.com/prometheus/prometheus/pull/1799

### CODE REVIEW

File: content/docs/querying/functions.md [hunk 1]
```diff
@@ -336,3 +325,4 @@ over time and return an instant vector with per-series aggregation results:
 * `max_over_time(range-vector)`: the maximum value of all points under the specified interval.
 * `sum_over_time(range-vector)`: the sum of all values under the specified interval.
 * `count_over_time(range-vector)`: the count of all values under the specified interval.
+* `quantile_over_time(scalar, range-vector)`: the given quantile of the values under the specified interval.
```
Comment (beorn7) [inline content/docs/querying/functions.md]: As we did in the code itself, should we change "under" to a less weird preposition like "in" or "within" or "contained in"?

**Comment by beorn7** [comment]
:+1: but needs a rebase.

**Comment by beorn7** [comment]
One more thing: If we just say "the given quantile" or "the k quantile", people will probably think about q-quantiles as in https://en.wikipedia.org/wiki/Quantile , i.e. integer numbers etc, e.g. "the 100-quantile is called percentile".

I haven't yet found a good name for the fractional quantiles we use throughout. Sometimes I see it called "φ quantile", which I also used in the Prometheus doc, e.g. in the famous paper https://www.cs.ucsb.edu/~suri/psdir/ency.pdf , but it's definitely not a widely known term. Not sure what's the best way here.

**Comment by beorn7** [comment]
OK, checked our existing documentation, I think we should do it in the same way as for https://prometheus.io/docs/querying/functions/#histogram_quantile() , i.e. use the wording "φ-quantile (0 ≤ φ ≤ 1)". That should make clear what we mean and is also consistent throughout the docs.

File: content/docs/querying/functions.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/querying/functions.md]: Done

Comment (brian-brazil) [inline content/docs/querying/functions.md]: Actually that was just an artifact of not being rebased.

**Comment by beorn7** [comment]
Perhaps the next-release branch should be reset to the top of master first?

**Comment by beorn7** [comment]
If you rebase _now_, things should be good.

And please consider what I said about the quantile terminology.

**Comment by brian-brazil** [comment]
That's updated.

**Comment by beorn7** [comment]
Thanks. :+1:

**Pull request merged**